### PR TITLE
forms

### PR DESCRIPTION
**Pull request by ChaabaneAnas**
-  Create a method called `current_user` in your ApplicationController to make current user data available to all controllers.
    -  It will return the first user from the database.
    -  Don't worry about proper authentication at this time. You will add it in the upcoming projects.
- Create forms to perform the following functions:
    - Creates a Post on behalf of the `current_user` (use the method that you created in your ApplicationController).
    - Create a comment on behalf of the `current_user` (use the method that you created in your ApplicationController).
- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    User.first
+  end
 end
```
Comment (codecaiine) [inline app/controllers/application_controller.rb:5]: - [x] From the project requirement. A user should be able to add a like to posts. Please create a likes controller in order to make your project works as expected :+1:

File: app/views/posts/_post.html.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  <div class="mb-3">
+      <%= f.label :title, class:"form-label" %>
+      <%= f.text_field :title, class:"form-control", placeholder:"Your post title here" %>
+  </div>
+  <div class="mb-3">
+    <%= f.label :text, class:"form-label" %>
+    <%= f.text_area :text, size: '50x10', class:"form-control", placeholder:"Your post body here" %>
+    </div>
+  <%= f.submit "Post",class:"btn btn-primary" %>
+<% end %>
```
Comment (codecaiine) [inline app/views/posts/_post.html.erb:19]: - [ ] Your project run well and I am able to create a user from the rails console but I can't see the forms created :cry: Please make sure to populate your database in order to see the form created and your project fit the given [wireframes](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLA3SVPWT6MXQ3C2OGFLBHIHES). Also make sure that we are able to comment a post and also like it :+1:

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @ChaabaneAnas , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good project setup :heavy_check_mark: 
- No rubocop errors :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- **I would like to strongly recommend you make your commit messages more meaningful in the future.** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (ChaabaneAnas) [reply app/controllers/application_controller.rb:5]: That's true i didn't see the requirement I'm going to work on it right now 
thanks for pointing to it

File: app/views/posts/_post.html.erb (hunk 2 above)
Comment (ChaabaneAnas) [reply app/views/posts/_post.html.erb:19]: @codecaiine Hello there,
Sorry, i couldn't understand what your pointing to here:
do you mean just adding the link_to ? (hyperlinks)  .

**Review by Munsa1 — APPROVED** [review]
Hi @ChaabaneAnas, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**